near/contracts/ft/src/lib.rs

```diff
@@ -14,6 +14,7 @@ use {
         },
         collections::LazyOption,
         env,
+        utils::assert_one_yocto,
```

Comment:
I'm not familiar with what a yocto is?

Reply:
one yocto is the smallest possible amount you can transfer